images/style.css

```diff
@@ -0,0 +1,114 @@
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #484848;
+  font-size: 14px;
+  font-family: Circular, "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
```

Comment:
- [ ] I would suggest you please use the font provided by microverse which is `font-family: 'Poppins'`. Due to this your content, and paragraph is looking differently.